src/raftstore/store/util.rs

```diff
@@ -177,22 +180,118 @@ pub fn get_region_approximate_size(db: &DB, region: &metapb::Region) -> Result<u
     Ok(size)
 }
 
+/// Lease records an expired time, for examining the current moment is in lease or not.
+/// A lease may be safe or be unsafe.
+// TODO: add a remote Lease.
```

Comment:
What's remote lease?

Reply:
A special lease that derives from `Lease`, it will be sent to the local read thread, so name it remote. If `Lease` expires, the remote must expires too.

Reply:
Please add explanation.